minesweepr.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Minesweepr Game</title>
+  <style>
+  table, td {
+    background: #DCDCDC;
+    border: 1px solid #808080;
+    border-collapse: collapse;
+    text-align: center;
+  }
+
+  td {
+    height: 20px;
```

Comment:
Is 20px based on other versions of the game? Or is this just what "looks right" when looking at the board?